### DISCUSSION TITLE
Pure Function

### DISCUSSION THREAD

**Discussion by r17x** [r17x/rin.rocks#10] [Q&A]
Kamu tahu apa yang musti kamu lakukan disini (asumsinya, kamu sudah membaca tulisan ini https://rin.rocks/kemurnian-fungsi)

**Comment by rgrahardi** [comment]
bang rin di contoh kodenya ada 2 functionkan, nah pertanyaannya yang mana yang pure function ?

**Reply by r17x** [reply]
@rgrahardi  2-2nya pure function, karena fungsi yang kedua tetap saja memakai fungsi pure didalamnya.
